Packs/WootCloud/Integrations/WootCloud/WootCloud_description.md

```diff
@@ -0,0 +1,454 @@
+# How to configure an API account
+To configure an API account:
+
+1. Signup with WootCloud to deploy a solution.
+www.wootcloud.com
+2. All WootCloud customers have access to the WootCloud API
+All WootCloud API base is:  https://api.wootuno.wootcloud.com/v1/
+
+All API calls need basic authentication, provided as Authroization HTTP header.
+
+API Keys
+Key Details
+"client_id": <client-key-here>
+"secret_key": <client-secret-key>
+
+
+Key Management APIs
```

Comment:
You can remove all the commands from here.
In the description, should only be how to configure the api